libs/barista-components/sunburst-chart/src/sunburst-chart.util.ts

```diff
@@ -54,8 +54,6 @@ export interface DtSunburstChartTooltipData {
   depth: number;
   /** Color for this node in this state */
   color: DtColors | string;
-  /** Color for this node when hovering in this state */
-  colorHover: DtColors | string;
```

Comment:
To have no breaking api change - we cannot just remove this - we need to deprecate it first and annotate it to be removed in the next major version.